src/components/Header.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { Box, Button } from '@chakra-ui/react';
+import { graphql, useStaticQuery } from 'gatsby';
+import Img from 'gatsby-image';
+
+import styles from '../css/Header.module.css';
+
+function Header(): JSX.Element {
+  const data: GatsbyTypes.HeaderQueryQuery = useStaticQuery<GatsbyTypes.HeaderQueryQuery>(graphql`
+    query HeaderQuery {
+      contentfulAsset(title: { eq: "Logo" }) {
+        fluid(quality: 100) {
+          ...GatsbyContentfulFluid
+        }
+        description
+      }
+    }
+  `);
+
+  return (
+    <div className={styles.navigation}>
```

Comment:
I would replace this with a `Flex` component (and use style props to style it)

For example:
```suggestion
    <Flex h="20vh" maxH="100px" wrap="nowrap" py="10px" px="40px" flexDir="row">
```

Then we can use `<Spacer />` components between each of the children to space them out